relay/nakama/allowlist_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/heroiclabs/nakama-common/api"
+	"github.com/stretchr/testify/mock"
+	"gotest.tools/v3/assert"
+	"pkg.world.dev/world-engine/relay/nakama/mocks"
+)
+
+func TestAllowList_UserIDRequired(t *testing.T) {
+	enableAllowListForTest(t)
```

Comment:
nit: we could make use of a `testify.Suite` and implement the `SetupTest` which would run some logic before each test in the suite.

see example here: https://github.com/Argus-Labs/world-engine/blob/cf1f24695b7d9bfd7d85c225d933843d93470972/evm/precompile/router/router_test.go